scripts/release/create-docs-prs.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+UTILS_PATH="$SCRIPT_DIR/../utils.sh"
+
+CLA_SIGNED_GITHUB_EMAIL="it-admin+mapboxci@mapbox.com"
+
+TMP_ROOT=$(mktemp -d)
+WORKTREE_TO_REMOVE=""
+cleanup() {
+    if [[ -d $WORKTREE_TO_REMOVE ]]
+    then
+        git -C "$SCRIPT_DIR" worktree remove "$WORKTREE_TO_REMOVE" --force
+    fi
+
+    rm -rf "$TMP_ROOT"
+}
+trap cleanup INT TERM HUP EXIT
+
+# shellcheck source=../utils.sh
+source "$UTILS_PATH"
+
+main() {
+    # Check that version env exists
+    [ -n "$VERSION" ]
+
+    step "Update mapbox/maps-ios@publisher-staging"
+    maps_ios_upload_docs
+
+    step "Open mapbox/maps-ios@publisher-production PR"
+    maps_ios_production_docs_pr
+
+    step "Update mapbox/ios-sdk"
+    ios_sdk_update_versions
+
+    step "Open mapbox/ios-sdk PR"
+    ios_sdk_open_pr
+
+    exit 0
+}
+
+maps_ios_upload_docs() {
+    info "Checkout publisher-staging worktree"
+    local staging_docs_path="$TMP_ROOT/docs-staging"
+    git worktree add "$staging_docs_path" "publisher-staging" --quiet
```

Comment:
the fact that we're always using publisher-staging and force pushing it means we can never run more than 1 release at a time. that would have been a problem today when I was patching 10.0 and 10.1 simultaneously. Is there a way for us to make this work better when running releases in parallel?

Reply:
Ah, I see the problem. Do we expect it to be a problem in the future? Should we support parallel releases?
As a workaround I can make a version-based PRs but that means we would lost an option to preview docs in the staging env.

Reply:
What if we do a hybrid — make version-based-branches for PRs, but also update publisher-staging to match those branches. That way, in the normal case, we still get previews, and in the parallel-releases case, latter PRs don't clobber previous ones.

Reply:
Done